nodeinstaller/internal/constants/testdata/expected-configuration-clh-snp.toml

```diff
@@ -12,34 +11,31 @@ enable_annotations = ['enable_iommu', 'virtio_fs_extra_args', 'kernel_params']
 enable_debug = false
 igvm = '/share/kata-containers-igvm.img'
 image = '/share/kata-containers.img'
-kernel_params = ''
+kernel = '/opt/kata/share/kata-containers/vmlinux.container'
+kernel_params = 'cgroup_no_v1=all systemd.unified_cgroup_hierarchy=1'
 path = '/bin/cloud-hypervisor-snp'
 rootfs_type = 'ext4'
-sev_snp_guest = true
-shared_fs = 'none'
-snp_guest_policy = 196608
+shared_fs = 'virtio-fs'
 valid_hypervisor_paths = ['/bin/cloud-hypervisor-snp']
-valid_virtio_fs_daemon_paths = ['/opt/confidential-containers/libexec/virtiofsd']
+valid_virtio_fs_daemon_paths = ['/opt/kata/libexec/virtiofsd']
 virtio_fs_cache = 'auto'
 virtio_fs_cache_size = 0
-virtio_fs_daemon = '/opt/confidential-containers/libexec/virtiofsd'
+virtio_fs_daemon = '/opt/kata/libexec/virtiofsd'
 virtio_fs_extra_args = ['--thread-pool-size=1', '--announce-submounts']
 virtio_fs_queue_size = 1024
 
 [Agent]
 [Agent.kata]
-dial_timeout = 90
-
-[Image]
-service_offload = false
```

Comment:
needed

Reply:
Sorry, I can't make it out from the GH comment - does this refer to dial_timeout as well, or just service_offload?

Reply:
If you look above the code, it says "Comment on lines 31 to 34" :) so both options are meant